### ISSUE TITLE
investigate supporting the atom-ide-ui plugin(s)

### ISSUE THREAD

**Issue by devoncarew** [dart-atom/dartlang#1158]
Some new plugins have been released that support IDE level UI features (atom-ide-ui). We should investigate supporting them for this plugin. Some features include diagnostics, outline view, jump to definition, find usages ,...

**Comment by devoncarew** [comment]
(/cc @cdavidjean)

**Comment by cdavidjean** [comment]
:(  I just moved them into dockable panels :)
It looks worth investigating.  The less code we have in the plugin, the less to maintain.  But it might increase breakages?

**Comment by devoncarew** [comment]
Yeah, not clear, but worth investigating. There would be less UI code for this package to maintain, and perhaps more familiarity if people do end up using `atom-ide-ui` for one or more other languages.

**Comment by cdavidjean** [comment]
I'll install it and play with it doing typescript tomorrow and get back to you.

**Comment by akonwi** [comment]
I'd like to see this plugin work with the `atom-ide-*` features because it allows for a consistent ux for a user across languages. If you don't mind, I could start a PR for implementing the integrations.
